docs/subsystems/typing-indicators.md

```diff
@@ -133,7 +133,7 @@ like the following:
 - `add_typist`
 - `remove_typist`
 - `get_group_typists`
-- `get_all_typists`
+- `get_all_pms_typists`
```

Comment:
We renamed "private message" to "direct messages", so this name is stale. We've also generally been trying to avoid abbreviations across the codebase; I think we can go with `get_all_direct_message_typists`.